src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -470,6 +470,10 @@ public BridgeChannelMessage selectedEndpoints(SelectedEndpointsMessage message)
     {
         Set<String> newSelectedEndpoints = new HashSet<>(message.getSelectedEndpoints());
 
+        if (logger.isDebugEnabled())
+        {
+            logger.debug("Selected " + newSelectedEndpoints);
```

Comment:
You can now do `logger.debug(() -> "Selected " + newSelectedEndpoints);` and avoid the extra `if` check.

Reply:
Doesn't this particular syntax create an anonymous class type (once) and a new instance every time it runs? Makes me wonder if simply concatenating the two strings is more efficient.